frontend/views/components/slider-circular/components/CircleSlider.vue

```diff
@@ -1,331 +0,0 @@
-<template lang='pug'>
```

Comment:
Hmm, instead of deleting all of these potentially useful components, making it difficult to find them say 1 year from now, can you move all these deleted files into `historical/` in case we decide we'd like to bring them back? It will be much more time consuming to find them in the git history.

Reply:
If we decide to do a UI similar to that in the future, I hope they do it from scratch because the current ones are not very usable or accessible.

Why do you think it is time-consuming to find them on git history? We have a very clean git history. Everything is documented. Moving old files to a `historical/` folder goes against the whole concept of git and only creates noise in the repo. 

Maybe a middle step to avoid redundancy: Have a single file `historical.md` or a github issue "Code Archive" with permalinks pointing to the respective file. For example:

##### Stale components
- [CircleSlider](https://github.com/okTurtles/group-income-simple/blob/028c9c8ee6d7531f73f0b0fd5d395b9caa8971ce/frontend/views/components/slider-circular/components/CircleSlider.vue)

Reply:
OK, that works, feel free to create that file! 👍